package.json

```diff
@@ -23,40 +25,50 @@
     "test:e2e:open": "npm-run-all clean build --parallel --race start:e2eserver cy:open",
     "licence:check": "license-check-and-add --f ./linting/license-check-and-add.config.json check",
     "licence:add": "license-check-and-add --f ./linting/license-check-and-add.config.json add",
-    "lint": "npm-run-all 'lint:allfiles -- .' 'lint:js -- .' 'lint:ts -- .' 'lint:styling -- \"**/*.(s)css\"' 'lint:format -- .'",
+    "lint": "npm-run-all 'lint:allfiles -- .' 'lint:ts -- .' 'lint:styling -- \"**/*.(s)css\"' 'lint:format -- .'",
     "lint:allfiles": "npm-run-all 'licence:add -- {1}' --",
-    "lint:js": "eslint --config ./linting/eslint.js.config.js --fix --max-warnings 0 --ignore-path ./linting/.eslintignore",
-    "lint:ts": "eslint --config ./linting/eslint.ts.config.js --fix --max-warnings 0 --ignore-path ./linting/.eslintignore",
+    "lint:ts": "eslint --config ./linting/eslint.config.js --fix --max-warnings 0 --ignore-path ./linting/.eslintignore",
```

Comment:
Being picky, as we now have a single config for all, can this be changed to not be file type specific - `lint:src` for instance? (With doc updates to match)